### PR TITLE
Update managing accounts guide

### LINKED ISSUE #1244: Create explanations about accounts
### Description
- Explain difference between creating credentials of an account, using the account on blockchain and value stored on blockchain
- Converting the address between different formats, with examples of all possible ways to do it

### Additional information
As of v6, the state is not stored based on address/account. Each module have access to generic key-value store.

---
"binaryAddress is for your use, for activating forging. address is the one you give to others to receive lisk or to be voted

---
It is always best to use the alphanumeric format(lsk....). Hex format(binaryaddress) has NO CHECKSUM 
lsk_format = base32(hex_format + checksum) 

---
Initially we were calling it binary address, which we realized not very correct.
Now, we call
* address => concept of address which is used in the code. it's 20 bytes long
* hex encoded address => hex represented 20 bytes address (this was binary address)
* lisk32 encoded address => address with checksum encoded with lisk32 format

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1244

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 1]
```diff
@@ -2,22 +2,173 @@
...
+:url_core_cli: v4@lisk-core::core-cli.adoc
 :url_sdk_cli: v6@lisk-sdk::client-cli.adoc
+:url_sdk_keyscreate: {url_sdk_cli}#keyscreate
+//Footnotes
+:fn_bip173: footnote:bip173[See {url_github_bip_173}[BIP173^]]
 
-The most convenient way to manage your account is to employ a user interface, such as Lisk Desktop.
-However, it is possible to manage your account directly from the command-line with the CLI of a Lisk node.
+====
+On this page, you'll learn how to  manage your account directly from the command-line with the CLI of a Lisk node.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
On this page, you'll learn how to manage your account directly from the command-line with the CLI of a Lisk node.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 2]
```diff
@@ -2,22 +2,173 @@
...
 
-The most important interactions are described below.
+Specifically, you'll learn about:
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: Imho, as we have 2 instances of "you'll learn" right after each other, it would sound better if we rephrased line 28 to read as follows:
Specifically, the following topics below will be covered in detail:

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 3]
```diff
@@ -2,22 +2,173 @@
...
 
-//TODO: Add link back once the page is updated
-//TIP: For more information about the available Lisk Core commands, check the CLI references for xref:{url_core_cli}[Lisk Core] and xref:{url_sdk_cli}[Lisk SDK].
-TIP: For more information about the available Lisk Core commands, check the CLI references for Lisk Core and xref:{url_sdk_cli}[Lisk SDK].
+* [x] The different credentials for an account
+* [x] How to create new account credentials
+* [x] How to create multiple accounts with a single passphrase
+* [x] How a new account is initialized on the blockchain
+* [x] Methods to manage account credentials
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
* [x] Methods used for managing account credentials
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 4]
```diff
@@ -2,22 +2,173 @@
...
+====
 
-== Creating a new account
+TIP: If you prefer a graphical user interface to manage your account, it is possible to use supported wallets like {url_lisk_wallet}[Lisk Wallet^].
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
TIP: If you prefer a graphical user interface to manage your account, it is possible to use supported wallets such as {url_lisk_wallet}[Lisk Wallet^].
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 5]
```diff
@@ -2,22 +2,173 @@
...
+|"m/25519'/134'/0'/0'"
+|The derivation path for the <<generator-keys>>.
+
+|`generatorKey`
+|e59f2f603b84b9cd91e2
+67240a342ba09adaad348e
+b984cc6e21c14fd02f28e7
+a|The public <<generator-keys,generator key>>.
+[#generator-keys]
+An extra {url_wiki_Ed25519}[Ed25519^] key pair, the generator keys, is added to a validator account to sign blocks.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc:92]:
```suggestion
An extra {url_wiki_Ed25519}[Ed25519^] key pair, namely the generator keys, are added to a validator account to sign blocks.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 6]
```diff
@@ -2,22 +2,173 @@
...
+A validator node only requires access to the private generator key, but not to the private key that is used for signing transactions.
+This mitigates the risk, that if an attacker is able to get the encrypted passphrase and the password used for encryption, the attacker has full control over the validator account.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: imho, we should reword this statement as follows:
"This mitigates the risk, whereby an attacker could acquire the encrypted passphrase and password, and gain full control over the validator account."

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 7]
```diff
@@ -2,22 +2,173 @@
...
+fab2c6e71dbbbd81c9f221
+70eb073cd4084dd15884fe
+59f2f603b84bac491e2672
+40a3425a09adaad348b984
+cc6e21c14fd02f28e7e
+|The private <<generator-keys,generator key>>.
+
+|`blsKeyPath`
+|"m/12381/134/0/0"
+|Similarly to the specifications of {url_eip2334}[EIP 2334^], Lisk uses the path `m/12381/134/chainIDConverted/0` for the BLS key derived for a chain, where `chainIDConverted = bytesToUint(chainID)`.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc:107]:
```suggestion
|Similarly to the specifications of {url_eip2334}[EIP 2334^], Lisk uses the path `m/12381/134/chainIDConverted/0` for the BLS key derived for a chain, where the `chainIDConverted = bytesToUint(chainID)`.
```

Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc:107]:
```suggestion
|Similarly to the specifications of {url_eip2334}[EIP 2334^], Lisk uses the path `m/12381/134/chainIDConverted/0` for the BLS key derived for a chain, where the `chainIDConverted = bytesToUint(chainID)`.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 8]
```diff
@@ -2,22 +2,173 @@
...
+
+|`blsKey`
+|95c7a571ecf569f91fc99
+5f9a8dba2e70904cb0793
+50372bc6d8a2b698678af
+6d4589ca9fffefd952deb
+c96a17317e18
+a|The public <<bls-keys,BLS key>>.
+[#bls-keys]
+An extra key pair, the BLS keys, is added to a validator account, following the key derivation method described in {url_eip2333}[EIP 2333^].
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc:117]:
```suggestion
An extra key pair, the BLS keys are added to a validator account, following the key derivation method described in {url_eip2333}[EIP 2333^].
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 9]
```diff
@@ -2,22 +2,173 @@
...
+BLS keys are used by validators to create aggregated signatures, among other things, for commits in Lisk blockchains.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
BLS keys are used by validators to create aggregated signatures, among other functions for commits in Lisk blockchains.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 10]
```diff
@@ -2,22 +2,173 @@
...
+9c6646e751517c514c70d
+11ec4ec4846310f5f7cdd
+edd598ade849496bcfe15
+d3baa554d06b148a2e9f2
+d349e2010df2a17d7a3d5
+7f13ab9bf290042c552d4a
+7d925c0877596ca8865b9c
+6ba2972eea5ce7831b014c
+|The proof that a validator owns the private key for the corresponding BLS public key.
+The proof-of-possession is important to prevent "rogue key" attacks, where attackers forge aggregate signatures on behalf of other participants.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The proof-of-possession is important to prevent "rogue key" attacks, whereby attackers forge aggregate signatures on behalf of other participants.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 11]
```diff
@@ -2,22 +2,173 @@
...
+|===
+
+== Key pair and address creation
+
+A *private key* is derived, as defined in {url_rfc8032}[RFC 8032^], from a {url_github_bip_39}[BIP 39^] *mnemonic passphrase* and a <<key-derivation-path>>.
+
+.Generating multiple private keys from 1 passphrase with key derivation
+image::run-blockchain/key_derivation.png[addressCreation]
+
+NOTE: Any of the child keys could be used to create an {url_wiki_Ed25519}[Ed25519^] private key, but by default, only the child key at the end of the path is used to create the Ed25519 private key.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
NOTE: Any of the child keys could be used to create a {url_wiki_Ed25519}[Ed25519^] private key. However, by default only the child key at the end of the path is used to create the Ed25519 private key.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 12]
```diff
@@ -2,22 +2,173 @@
...
+
+=== Key Derivation path
+
+The *key derivation path* allows to derive a specific key from within a tree of keys.
+
+The derivation path is an array of non-negative integers smaller than 2^32.
+To make it easier to read a derivation path, they are often written in a form that omits a large constant.
+For example, the path 44'/134'/0' (notice the apostrophe after each number) represents the array [44+2^31, 134+2^31, 2^31].
+
+The whole derivation path is then written m/44'/134'/0', indicating that we start by deriving a master key before deriving successive child keys.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The whole derivation path is then written as m/44'/134'/0', indicating that we start by deriving a master key before deriving successive child keys.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 13]
```diff
@@ -26,66 +177,37 @@ Lisk Core::
...
-If you wish to create transactions without having the node running, use the `--offline` flag.
-These transactions need to be signed later with `transaction:sign`.
+=== Creating the other credentials
 
-For more information on this topic, check out this guide, xref:{url_sdk_guide_txoffline}[Creating transactions and signing them offline].
-====
+Next, use the xref:{url_sdk_keyscreate}[keys:create] command to generate all relevant account credentials:
 
-Start the node, if it is not running already:
+IMPORTANT: Creation of new account credentials always happens off-chain.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
IMPORTANT: The creation of new account credentials always happens off-chain.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 14]
```diff
@@ -94,67 +216,97 @@ Lisk Core::
...
 [source,bash]
 ----
-./bin/run start
+./bin/run keys:create
 ----
 --
 =====
 
-Transactions can be created by using the `transaction:create` command:
+When asked for the passphrase, enter the passphrase created in step <<creating-a-new-passphrase>>.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
When asked for the passphrase, enter the passphrase previously created in the step <<creating-a-new-passphrase>>.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 15]
```diff
@@ -94,67 +216,97 @@ Lisk Core::
...
-  ASSETID   Registered transaction asset id.
-  FEE       Transaction fee in Beddows.
+The encrypted credentials are used for example in the config, when the node needs account information to generate new blocks for a validator.
 
-OPTIONS
-  -a, --asset=asset                          Creates transaction with specific asset information.
-  -d, --data-path=data-path                  Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.
-  -j, --json                                 Print the transaction in JSON format
-  -n, --network=network                      [default: mainnet] Default network config to use. Environment variable "LISK_NETWORK" can also be used.
+TIP: If you don't wish to encrypt your credentials, it is possible to skip this step by adding the `--no-encryption` flag to the `keys:create` command.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
TIP: If you do not wish to encrypt your credentials, it is possible to skip this step by adding the `--no-encryption` flag to the `keys:create` command.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 16]
```diff
@@ -94,67 +216,97 @@ Lisk Core::
...
 
-  -p, --passphrase=passphrase                Specifies a source for your secret passphrase. Command will prompt you for input if this option is not set.
-                                             	Examples:
-                                             	- --passphrase='my secret passphrase' (should only be used where security is not important).
+The password will be required everytime the credentials need to be decrypted, for example when enabling block generation on a node.
+Therefore, the password should be saved and kept at a secure place, similar to the passphrase of the account.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Hence, it is advisable to store and maintain the password in a secure location similar to the passphrase of the account.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 17]
```diff
@@ -94,67 +216,97 @@ Lisk Core::
...
+====
 
-  --nonce=nonce                              Nonce of the transaction.
+== Managing multiple accounts with a single passphrase
 
-  --offline                                  Specify whether to connect to a local node or not.
+Key derivation allows multiple key pairs to be derived from the same secret passphrase as a tree of keys.
 
-  --pretty                                   Prints JSON in pretty format rather than condensed.
+This means, a user can manage multiple different accounts, while they only need to remember a single passphrase.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
This ensures that a user can manage multiple different accounts with ease, as they are only required to remember a single passphrase.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 18]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
 ----
 --
 =====
 
-.Example output
-[source,json]
-----
-{"transaction":"0802100018002080c2d72f2a200fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a32270880c2d72f121486555265f0110b4ed5a8cb95dbc732e77732c4741a0a73656e6420746f6b656e3a40c442cd1311808974d5f41a69dd25b2e3a516e463445a4c5eb292f0c7b7de0b33627a729da7969875d0f659fbe56f3e2ca947bc02873f8ac5a1753ac90fb15605"}
-----
+The same passphrase is used for every account, the only thing that will change is the <<key-derivation-path>>, for example when signing transactions.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The same passphrase is used for every account, the only change that will occur is the <<key-derivation-path>>, for example when signing transactions.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 19]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
-+
---
-[source,bash]
-----
-$ ./bin/run transaction:send 0802100018002080c2d72f2a200fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a32270880c2d72f121486555265f0110b4ed5a8cb95dbc732e77732c4741a0a73656e6420746f6b656e3a40c442cd1311808974d5f41a69dd25b2e3a516e463445a4c5eb292f0c7b7de0b33627a729da7969875d0f659fbe56f3e2ca947bc02873f8ac5a1753ac90fb15605
-----
---
-=====
+If you are using a custom key-derivation path, make sure to always specify the key derivation path when providing the passphrase.
+For example, for the `transaction:create` command, the key derivation path can be specified like so:
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
For example, for the `transaction:create` command, the key derivation path can be specified as shown in the example snippet below:
```

Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
For example, for the `transaction:create` command, the key derivation path can be specified as shown in the example snippet below:
```

Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
For example, for the `transaction:create` command, the key derivation path can be specified as shown in the example snippet below:
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 20]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+To create a new account in the blockchain network, it is necessary to initialize the account.
+
+A new account is initialized by sending tokens to an address for the first time.
+
+When tokens are sent to a new address, it is required to pay an *account initialization fee*.
+
+In the Lisk Mainnet, the account initialization fee is 0.05 LSK.
+
+The account initialization fee has to be paid on top of the other fees for a transaction.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
In addition to the transaction fees, users are required to pay an account initialization fee.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 21]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+After the address received first tokens, the address is stored in the user substore.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Once the address has received its initial tokens, it is then saved in the user substore.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 22]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+=== User Substore
+The user substore is a data store of the Token module and contains entries dedicated to storing the balances of users for a given address and tokenID.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: imho I would suggest we rephrase this sentence to the following:
The Token module features a user substore that comprises entries exclusively intended for storing user balances for a specific address and tokenID.

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 23]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
 
-ARGUMENTS
-  TRANSACTION  A transaction to be sent to the node encoded as hex string
+The user substore contains entries with:
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The user substore contains the following entries:
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 24]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+}
 ----
 ====
+
+== Account address formats
+
+The account address can be displayed in different formats, depending on the context.
+
+=== Hexadecimal representation
+The hexadecimal representation of the address is directly derived from the 20 first Bytes of the public key.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The hexadecimal representation of the address is directly derived from the first 20 bytes of the public key.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 25]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+The address in hex format is often used internally as parameter.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The address in hex format is often used internally as a parameter.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 26]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+.Hex-format address example
+ 0xc247a42e09e6aafd818821f75b2f5b0de47c8235
+
+==== Hex address computation
+To compute the address in hex format, the following information is required:
+
+* account public key, e.g. 0x0eb0a6d7b862dc35c856c02c47fde3b4f60f2f3571a888b9a8ca7540c6793243
+
+First, create the SHA-236 hash of the public key
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
First, create the SHA-236 hash of the public key:
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 27]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+Each address starts with the prefix “lsk" and a checksum is appended.
+
+.Lisk32 address example
+ lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu
+
+The *Lisk32 representation* displays addresses as a checksum with the "lsk" prefix to prevent accidental typing mistakes.
+
+==== Lisk32 address computation
+The procedure to obtain the Lisk32 representation of an <<hexadecimal-representation>> of an address is as follows:
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The procedure to obtain the Lisk32 representation of a <<hexadecimal-representation>> of an address is as follows:
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 28]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+. The <<hexadecimal-representation,hex address>> is used to create a 30-bit checksum of the address, using a BCH code{fn_bip173}.
+This step provides protection against accidental typing mistakes.
+. Concatenate the address and the output of step 1.
+. Encode the output of step 2 in a custom Base32 format, see {url_github_lip_18}[LIP 0018^] for details.
+Lower-case letters and digits are used, the characters i, l, 1 and 0 are removed for usability reasons.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Lower-case letters and digits are used, and to improve usability the characters i, l, 1 and 0 have been excluded.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 29]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+. Add the prefix "lsk" to the output of step 3.
+
+NOTE: The Lisk32 representation always tarts with "lsk" and the final address is 41 characters long.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
NOTE: The Lisk32 representation always starts with "lsk" and the final address consists of an amount of 41 characters.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 30]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+There are two recommended ways to manage the account keys, depending on the use case:
+
+.Account credentials
+[cols="1,2",options="header",stripes="hover"]
+|===
+|Key management method
+|Use Case
+
+|Using a secret passphrase
+|For users that need their key pair only on local machines, e.g., for singing transactions.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
|For users that need their key pair only on local machines, e.g., for signing transactions.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 31]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+Requires to remember or store secretly a secret recovery phrase and a password, but does not require to store and backup any encrypted data locally.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]: imho we should reword this as follows:
It is necessary to securely store or remember a secret recovery phrase and password. However, it is not necessary to back up and store any encrypted data locally.

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 32]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+
+|Storing the encrypted secret key
+|For users that need the key pair only on some remote server, e.g., a validator node on a remote data center.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
|For users that solely require the key pair on a remote server, e.g., a validator node on a remote data center.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc [hunk 33]
```diff
@@ -163,77 +315,153 @@ Lisk Core::
...
+Requires to store and backup an encrypted file, but needs to remember or secretly store only one password.
```
Comment (chris529) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Storing and backing up an encrypted file is necessary, but the user only needs to remember or securely store one password.
```

**Review by chris529 — COMMENTED** [review]
Looks great! Just a few minor points 👍

**Review by chris529 — COMMENTED** [review]
Looks great! Just a few minor points 👍

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 2 above)
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
Specifically, the following topics below will be covered in detail:
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 5 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc:92]: Imho it is correct to use "is" here, because it refers to a pair of keys.

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 6 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
This mitigates the risk, whereby an attacker could acquire the encrypted passphrase and the password used for encryption, and gain full control over the validator account.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 9 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
BLS keys are used by validators to create aggregated signatures for commits in Lisk blockchains, among other functions.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 20 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
NOTE: The account initialization fee has to be paid on top of the other fees for a transaction, when tokens are sent to a new address for the first time.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 22 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
The Token module features a user substore that comprises entries exclusively intended for storing user balances for a specific address and tokenID.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 31 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc]:
```suggestion
It is necessary to securely store or remember a secret recovery phrase and password. However, it is not necessary to back up and store any encrypted data locally.
```

File: docs/modules/ROOT/pages/run-blockchain/account-management.adoc (hunk 5 above)
Comment (chris529) [reply docs/modules/ROOT/pages/run-blockchain/account-management.adoc:92]: Yes, fair point, I was thinking it was keys, however as you say as it is referring to a single pair of keys then in this instance it is a singular noun phrase, and therefore is the subject of the sentence. So in this case the verb "is" would be appropriate as that agrees with the singular subject "key pair". 👍

**Review by chris529 — APPROVED** [review]
LGTM 👍

**Review by mosmartin — APPROVED** [review]
🚀
